config/ember-try.js

```diff
@@ -2,12 +2,6 @@
 module.exports = {
   scenarios: [
     {
-      name: 'default',
```

Comment:
not sure what this was for since tests are running against `release`, `beta` and `canary`